src/CoolProp.jl

```diff
@@ -10,6 +10,7 @@ using CoolProp_jll
 errcode = Ref{Clong}(0)
 const buffer_length = 20000
 message_buffer = Array{UInt8}(undef, buffer_length)
+const maxN = 100 #max number of species per model
```

Comment:
I am wondering if anyone has a suggestion for the value of `maxN`. Is there any known defined limit for the maximum number of species in a model?